modules/extensions-support/src/main/java/org/mule/module/extension/internal/runtime/BaseObjectBuilder.java

```diff
@@ -25,7 +25,7 @@
  *
  * @since 3.7.0
  */
-abstract class BaseObjectBuilder<T> implements ObjectBuilder<T>
+public abstract class BaseObjectBuilder<T> implements ObjectBuilder<T>
```

Comment:
to better separate implementations of this class in more meaningful packages instead of having them all together in a big cat bag